examples/cop-dem-glo-30.json

```diff
@@ -0,0 +1,97 @@
+{
+  "type": "Collection",
+  "id": "cop-dem-glo-30",
+  "stac_version": "1.0.0",
+  "description": "The Copernicus DEM is a Digital Surface Model (DSM) which represents the surface of the Earth including buildings, infrastructure and vegetation. We provide two instances of Copernicus DEM named GLO-30 Public and GLO-90. GLO-90 provides worldwide coverage at 90 meters. GLO-30 Public provides limited worldwide coverage at 30 meters because a small subset of tiles covering specific countries are not yet released to the public by the Copernicus Programme. Note that in both cases ocean areas do not have tiles, there one can assume height values equal to zero. Data is provided as Cloud Optimized GeoTIFFs and comes from Copernicus DEM 2021 release.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "./cop-dem-glo-30.json",
+      "type": "application/json",
+      "title": "Copernicus DEM GLO-30"
+    },
+    {
+      "rel": "handbook",
+      "href": "https://object.cloud.sdsc.edu/v1/AUTH_opentopography/www/metadata/Copernicus_metadata.pdf",
+      "type": "application/pdf",
+      "title": "Copernicus DEM User handbook",
+      "description": "Also includes data usage information"
+    },
+    {
+      "rel": "license",
+      "href": "https://spacedata.copernicus.eu/documents/20126/0/CSCDA_ESA_Mission-specific+Annex.pdf",
+      "type": "Copernicus Data Access"
+    },
+    {
+      "rel": "self",
+      "href": "/workspaces/cop-dem/cop-dem-glo-30/cop-dem-glo-30.json",
+      "type": "application/json"
+    }
```

Comment:
For examples, we usually exclude the `self` href, since it's an absolute path that won't make sense on most systems (e.g. this one's for the dev container, but doesn't make sense when accessed via its raw Github href).

`include_self_href=False` could be enabled via a CLI option, but I've often found it easier to make a standalone script to generate the examples, e.g. https://github.com/stactools-packages/usda-cdl/blob/main/scripts/create_examples.py.

Reply:
fixed by adding a script that normalized as you suggested